global.json

```diff
@@ -1021,6 +1021,30 @@
         ]
       },
       {
+        "name": "REST Client",
+        "priority": 0.883,
+        "id": "rest-client",
+        "docs": "https://raw.githubusercontent.com/feedhenry-templates/fh-api-mapper/master/README.md",
+        "type": "other",
+        "image": "public/img/cloud_plugins/fh.png",
+        "category": "Platform Service",
+        "description": "Explore APIs which may be protected by a firewall, or only accessible to the cloud via VPN. Supports saving requests, and generates useful code snippets.",
+        "appTemplates": [
+          {
+            "id": "rest-client-service",
+            "name": "Barcode Reader",
+            "type": "cloud_nodejs",
+            "repoUrl": "git://github.com/feedhenry-templates/fh-api-mapper.git",
+            "githubUrl": "https://github.com/feedhenry-templates/barcode-reader-service.git",
```

Comment:
this seems like copy&paste error, and may be cause of a failure to deploy this service, details here: https://github.com/fheng/fhcap/pull/1065#issuecomment-143977481